functions/test/contract/token-api-contract.test.ts

```
@@ -0,0 +1,317 @@
+            errorHandler(err as Error, req, res, vi.fn());
+          }
+          finish();
+        };
+
+        try {
+          await tokenHandler.getTokenInfo(req, res, next);
+          finish();
+        } catch (err) {
+          next(err);
+        }
+      });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)